.github/workflows/tests.yml

```diff
@@ -1,12 +1,20 @@
 name: Tests
-on: [push, pull_request]
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - '*'
+
 jobs:
   tests:
     continue-on-error: ${{ matrix.experimental }}
     strategy:
       fail-fast: false
       matrix:
-        ruby: [2.5, 2.6, 2.7, '3.0', 3.1]
+        ruby: [2.7, 3.0, 3.1, 3.2]
```

Comment:
Could you revert `3.0` to `'3.0'`?

It is intentional. When `matrix.ruby` is number `3.0`, `${{ matrix.ruby }}` is converted to `3` and the latest version of ruby 3.x is picked up. See https://github.com/LeFnord/ruby-oci8/actions/runs/4222066763/jobs/7330111735#step:7:3 and https://github.com/LeFnord/ruby-oci8/actions/runs/4222066763/jobs/7330111735#step:7:26.
It must be string `'3.0'` to use the latest version of ruby 3.0.x.

Reply:
ah … sure, will revert it … and yeah it is really a strange behaviour, thanks for spotting it